contexts/authentication.tsx

```diff
@@ -157,7 +157,10 @@ export const AuthenticationProvider = ({ children }) => {
   }, [DAOService, wallet?.address]);
 
   useEffect(() => {
-    if (wallet && wallet?.address) updateWalletBalance();
+    if (!wallet?.address) return localStorage.setItem("currentWallet", "");
```

Comment:
@moshmage We can use `sessionStorage`, but we still need to set the value as "" in case of user disconnect the wallet manually using the `Disconnect Wallet` button